templates/controllers/grid/grid.tpl

```diff
@@ -74,8 +74,6 @@
 		{/if}
 		{foreach from=$gridBodyParts item=bodyPart}
 			{$bodyPart}
-		{foreachelse}
-			<tbody><tr></tr></tbody>
```

Comment:
I think this is here for the category grids with no category items, so it will be ready to receive any new item that could be added and retrieved by Ajax. But I am checking :)